dotnet/src/Connectors/Connectors.OpenAIV2/Core/ClientCore.cs

```diff
@@ -56,22 +56,22 @@ internal partial class ClientCore
     /// <summary>
     /// Identifier of the default model to use
     /// </summary>
-    internal string ModelId { get; init; } = string.Empty;
+    protected internal string ModelId { get; init; } = string.Empty;
```

Comment:
Is this still required?  Seems like you've mostly replaced usage of this with a model parameter into each method.

Reply:
Actually this is required, as the services uses it to call the ClientCore.